Formula/kubebuilder.rb

```diff
@@ -25,7 +25,7 @@ def install
       -X main.goos=#{goos}
       -X main.goarch=#{goarch}
       -X main.gitCommit=#{Utils.git_head}
-      -X main.buildDate=#{Time.now.iso8601}
+      -X main.buildDate=#{Time.at(ENV["SOURCE_DATE_EPOCH"].to_i).iso8601}
```

Comment:
Is it important to use `utc` everywhere? The timezone isn't set during the build, and I wonder if not using `utc` would be a problem (wrt reproducibility).

Reply:
Yea, let's do `utc`.